HW02/vitaly_gavrilenya/my_methods.rb

```diff
@@ -0,0 +1,34 @@
+module NewArrayMethods
+  refine Array do
+    def my_map
+      result_array = []
+      for i in 0...self.length
+        result_array << yield(self[i])
+      end
+      result_array
+    end
+
+    def my_select
+      result_array = []
+      for i in 0...self.length
+        yield(self[i]) ? result_array << self[i] : result_array
```

Comment:
Do you really need it? 🤔 
```suggestion
        result_array << self[i] if yield(self[i]) 
```